### PR TITLE
CVE-2020-1732

### PR DESCRIPTION
**Pull request by pedrohc**

### CODE REVIEW

File: 2020/1xxx/CVE-2020-1732.json [hunk 1]
```diff
@@ -4,15 +4,73 @@
...
+    },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "vendor_name": "Red Hat",
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "Soteria",
```
Comment (cristina479) [inline 2020/1xxx/CVE-2020-1732.json]: Is the affected product Soteria or WildFly as stated in the description?

File: 2020/1xxx/CVE-2020-1732.json [hunk 2]
```diff
@@ -4,15 +4,73 @@
...
+                                "version": {
```
Comment (cristina479) [inline 2020/1xxx/CVE-2020-1732.json]: The description says that versions **before** 19.0.0.beta3 are affected.

File: 2020/1xxx/CVE-2020-1732.json (hunk 1 above)
Comment (pedrohc) [reply 2020/1xxx/CVE-2020-1732.json]: Its soteria, but our fixes are in wildfly which I believe ships it inside. I'm working on getting that description improved.

File: 2020/1xxx/CVE-2020-1732.json (hunk 2 above)
Comment (pedrohc) [reply 2020/1xxx/CVE-2020-1732.json]: I've been informed that there is no upstream fix yet. That info was retracted after the PR was opened.

I'm not finding any references to help with this.

**Comment by pedrohc** [comment]
> Several changes need to be made in order to merge the PR:
> 
>     1. Improve description (the affected product is Soteria but the description can lead users into thinking the affected product is WildFly).
> 
>     2. Include affected version (the version info was retracted)
> 
>     3. URL link https://issues.redhat.com/browse/WFLY-13046 cannot be included as a reference because it is not publicly accessible due to insufficient permissions

Ok I think I got everything covered now.

1. Description updated to Soteria both here and in the referenced bug.

2. Version info corrected to the one as tagged in upstream commit. Both here and in the original bug description and fixed in fields.

3. This reference was removed and upstream commit link added.

Let me know if anything else is needed.

**Pull request merged**